### PR TITLE
change

### PR DESCRIPTION
**Pull request by liorpaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,5 @@
 **Design**:
 https://docs.google.com/document/d/1QLXDV6H0oFZnsC2ZjCHqcNocXrtAKdscEHNysotrRNk/edit
 
-periodic.py cron job should trigger
+periodic.py cron job should trigger events by time
```
Comment (rhosisey) [inline README.md]: asdawfsegdrrhgs

File: README.md [hunk 2]
```diff
@@ -3,5 +3,5 @@
...
 api.py handles slack commands
```
Comment (zeako) [inline README.md]: bad decision, who hired you?

File: README.md [hunk 3]
```diff
@@ -3,5 +3,5 @@
 **Design**:
 https://docs.google.com/document/d/1QLXDV6H0oFZnsC2ZjCHqcNocXrtAKdscEHNysotrRNk/edit
```
Comment (AssafMiz) [inline README.md]: it's a shitty code

File: README.md [hunk 4]
```diff
@@ -3,5 +3,5 @@
...
 
-periodic.py cron job should trigger
```
Comment (ishustri) [inline README.md]: !

**Pull request merged**